tests/gradients/test_finite_difference.py

```diff
@@ -189,11 +193,29 @@ def test_y0(self, mocker):
             qml.RY(-0.654, wires=[0])
             qml.expval(qml.PauliZ(0))
 
-        tapes, fn = finite_diff(tape, order=1)
+        tapes, fn = finite_diff(tape, approx=1)
 
         # one tape per parameter, plus one global call
         assert len(tapes) == tape.num_params + 1
 
+    def test_y0_provided(self, mocker):

```

Comment:
```suggestion
    def test_y0_provided(self):
```
Looks like this isn't used?

Reply:
I think this is a test I _tried_ to use it in, struggled, and, uh, gave up. I realized that `len(tapes)` is sufficient in terms of counting number of device executions.